translations/zh-hans.yaml

```diff
@@ -2167,6 +2167,39 @@ clusterNew:
       placeholder: 客户端秘钥
     region:
       label: 区域
+    aadClientAppId:
+      label: Client ID
+      placeholder: Your Client ID of Azure Active Directory.
```

Comment:
Remove the periods in all the new placeholders, some have them and some don't. We should be consistent and for place holders we typically don't use periods.